### PR TITLE
[ONNX] Support clamp_min and clamp_max

### PR DESCRIPTION
**Pull request by BowenBao**
clamp_min is used in `torch.nn.functional.normalize`. Update symbolic_opset11 to support with updated clip in onnx opset 11.

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -1515,6 +1515,15 @@ def forward(self, x, k):
         k = torch.tensor(3)
         self.run_test(MyModuleDynamic(), [x, k])
 
+    @skipIfUnsupportedOpsetVersion([7, 12])
```
Comment (neginraoof) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:1518]: Why is opset 12 skipped?

Comment (neginraoof) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:1518]: I guess we can enable this after ORT version is updated?

**Review by neginraoof — APPROVED** [review]

**Review by houseroad — APPROVED** [review]
Thanks
